pkg/task/restore.go

```diff
@@ -39,16 +44,28 @@ type RestoreConfig struct {
 
 	Online   bool `json:"online" toml:"online"`
 	NoSchema bool `json:"no-schema" toml:"no-schema"`
+
+	// MergeSmallRegionSizeBytes is the threshold of merging small regions (Default 96MB, region split size).
+	// MergeSmallRegionKeyCount is the threshold of merging smalle regions (Default 960_000, region split key count).
+	// See https://github.com/tikv/tikv/blob/v4.0.8/components/raftstore/src/coprocessor/config.rs#L35-L38
+	MergeSmallRegionSizeBytes uint64 `json:"merge-region-size-bytes" toml:"merge-region-size-bytes"`
+	MergeSmallRegionKeyCount  uint64 `json:"merge-region-key-count" toml:"merge-region-key-count"`
```

Comment:
include these into `adjustRestoreConfig`, otherwise before the next update, BR in SQL will use 0 for both of these (which is probably fine 🤔).